.travis.yml

```diff
@@ -15,47 +15,26 @@ python:
   - "2.7"
 node_js:
   - "0.10"
+env:
+  - RUN_TEST=jshint
+  - RUN_TEST=unittest
+  - RUN_TEST=uitest-phantom-ci
+  - RUN_TEST=uitest-slimer-ci
+  - RUN_TEST=test-langpacks
+  - RUN_TEST=test-package
 before_script:
+  - "export SLIMERJSLAUNCHER=$(which firefox)"
```

Comment:
I'd recommend using a specific FF version [1]. Here's a snippet as an example. Bear in mind Slimer only supports stable versions:

```yaml
addons:
  firefox: "18.0"
```

As I mentioned on irc previously - we're using 18 on payments at the moment because that's the oldest version that we still need to support (whilst it's older than 20 it seems to work with slimer ok). It means you'll catch issues like missing `window.location.origin` etc. Testing more than one version will probably make a lot of sense in due course assuming Travis can support that.

[1] http://docs.travis-ci.com/user/firefox/